src/components/Products/AddItemButton.js

```diff
@@ -0,0 +1,18 @@
+import { AiOutlineFileAdd } from 'react-icons/ai'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+function AddItemButton() {
+  return (
+    <Button>
+      <AiOutlineFileAdd />
+      <Link to="/product-form">Add Item</Link>
+    </Button>
+  )
+}
+
+const Button = styled.button`
+  font-size: large;
+`
```

Comment:
Same here! And this is almost the same component as the `BackButton`. We want to avoid repeating ourselves. Why not have a `CustomLink` component and add the `path` and `name` as props?
```suggestion
function CustomLink({path, name, icon}) {
  return (
    <>
      {icon}
      <LinkStyled to={path}>{name}</LinkStyled>
    </>
  )
}

const LinkStyled = styled(Link)`
  font-size: large;
`
```